src/libs/protocols/protos/agent.proto

```diff
@@ -293,8 +293,10 @@ message CreateSandboxRequest {
 	// This field, if non-empty, designates an absolute path to a directory
 	// that the agent will search for OCI hooks to run within the guest.
 	string guest_hook_path = 6;
+	// GuestHookTimeout is the number of seconds before aborting the hook, If set, it must be greater than zero.
+	int32 guest_hook_timeout = 7;
```

Comment:
Please just add new fields at the end of a struct. Inserting things in the middle will break grpc compatibility.